### ISSUE TITLE
Can't run on Windows -- Possible problem with leaflet?

### ISSUE THREAD

**Issue by eykamp** [thingsboard/thingsboard#293]
I'm working on Windows 10, and I'm following the instructions here:

    https://thingsboard.io/docs/user-guide/contribution/how-to-contribute/

I start the server like this:

    C:\dev\thingsboard\ui>mvn clean install -P npm-start

After a while, I get this:

    [INFO] ==> ≡ƒîÄ  Listening on port 3000. Open up http://localhost:3000/ in your browser.

So far so good.  Then I hit that URL in Chrome, and I get a huge stacktrace of errors  (see https://pastebin.com/CeHbsFEM for the whole thing).  The first one is this:

````
[ERROR] ERROR in ./~/leaflet/dist/leaflet.css
[ERROR] Module build failed: ModuleBuildError: Module build failed: Error: spawn UNKNOWN
[ERROR]     at exports._errnoException (util.js:1026:11)
[ERROR]     at ChildProcess.spawn (internal/child_process.js:313:11)
[ERROR]     at Object.exports.spawn (child_process.js:380:9)
[ERROR]     at module.exports.err (C:\dev\thingsboard\ui\node_modules\execa\index.js:139:26)
[ERROR]     at fsP.writeFile.then (C:\dev\thingsboard\ui\node_modules\exec-buffer\index.js:35:15)
[ERROR]     at DependenciesBlock.onModuleBuildFailed (C:\dev\thingsboard\ui\node_modules\webpack-core\lib\NormalModuleMixin.js:315:19)
[ERROR]     at nextLoader (C:\dev\thingsboard\ui\node_modules\webpack-core\lib\NormalModuleMixin.js:270:31)
[ERROR]     at C:\dev\thingsboard\ui\node_modules\webpack-core\lib\NormalModuleMixin.js:292:15
[ERROR]     at context.callback (C:\dev\thingsboard\ui\node_modules\webpack-core\lib\NormalModuleMixin.js:148:14)
[ERROR]     at C:\dev\thingsboard\ui\node_modules\img-loader\index.js:46:7
[ERROR]  @ ./src/app/widget/lib/openstreet-map.js 23:0-35
````
followed by a series of similar leaflet related errors.

Any idea how to resolve this?

**Comment by eykamp** [comment]
I still have no solution for this, but I am thinking it may be related to one or more missing node packages.  Is there a full list of node dependencies?

**Comment by eykamp** [comment]
I've resolved the issue.  I think it was a node configuration problem.  I deleted the entire ui/node_modules folder, then ran "npm install" and "npm install bufferutil utf-8-validate" from the ui folder, and while I am still getting errors in the cmd window, I can load the UI via the browser at http://localhost:3000.  So I guess it's sort of working?  

I'm going to close this, with the note that the node dependencies need to be documented for Windows.  I think there are some webpack modules I need to load to make my lingering errors go away, but for the moment I think this problem is fixed enough.
